src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductOptionRepository.php

```diff
@@ -40,4 +41,40 @@ public function findByName(string $name, string $locale): array
             ->getResult()
         ;
     }
+
+    /**
+     * @return ProductOptionInterface[]
+     */
+    public function findByNamePart(string $phrase, string $locale, int $limit = 15): array
```

Comment:
```suggestion
    public function findByPhrase(string $phrase, string $locale, int $limit = 15): array
```
Fits better, we're also checking the code